lib/rack/response.rb

```diff
@@ -95,11 +95,23 @@ def chunked?
       CHUNKED == get_header(TRANSFER_ENCODING)
     end
 
+    def enumerable_body?
+      @body.respond_to?(:each)
+    end
+
+    def streaming_body?
```

Comment:
This function seems unnecessary.  If you think we should add it, we should have specs for it.